draft-ietf-quic-recovery.md

```diff
@@ -636,7 +640,9 @@ The pacing rate is a function of the mode, the congestion window, and
 the smoothed rtt.  Specifically, the pacing rate is 2 times the
 congestion window divided by the smoothed RTT during slow start
 and 1.25 times the congestion window divided by the smoothed RTT during
-slow start.
+slow start.  In order to fairly compete with flows that are not pacing,
+it is recommended to not pace the first 10 sent packets when exiting
+quiescence.
```

Comment:
I wonder if this entire section should be removed and left to implementers. But this is fine for now, let's discuss that separately.